solr/modules/cross-dc/build.gradle

```diff
@@ -24,7 +24,7 @@ description = 'Cross-DC Replication Plugins'
 ext {
   // Use of the security manager is not supported on windows.
   // Mockito does not work with it enabled.
-  useSecurityManager = !Os.isFamily(Os.FAMILY_WINDOWS)
+  useSecurityManager = rootProject.ext.runtimeJavaVersion <= JavaVersion.VERSION_23 && !Os.isFamily(Os.FAMILY_WINDOWS)
```

Comment:
this is basically an addition which was used later.

Maybe change this to use `useSecurityManager &= !Os.isFamily(Os.FAMILY_WINDOWS)`

This would take the setting from top-level.

Reply:
Ah this needs to be done differently. I would do the `&` trick here:

https://github.com/apache/solr/blob/de09232c53a18a0391d7386fea21a2aaae03af66/gradle/testing/randomization.gradle#L207

Uwe

Reply:
Change it to `&=`here and it will work correctly also for possible other modules.

Reply:
Agreed, this is much better